Makefile

```diff
@@ -102,6 +113,72 @@ run-docker:
 		-p 8001:8001 \
 		hypothesis/lms:$(DOCKER_TAG)
 
+.PHONY: run-docker-dev
+run-docker-dev:
+	docker run \
+		--name lms \
+		--net h_default \
+		-v /Users/hmstepanek/Documents/work/dev/lms/lms:/var/lib/lms/lms \
```

Comment:
Mount the python code as a volume so changes get rebuilt without re-building the docker container.